server/src/test/resources/sql/table.iq

```diff
@@ -215,4 +215,8 @@ drop table if exists t;
 
 !update
 
+truncate table t;
```

Comment:
Can we have a positive test here? To tell the effectiveness of TRUNCATE

Reply:
I use more efficient methods to clear data and add one positive test.